dnf/cli/commands/repoquery.py

```diff
@@ -177,6 +177,8 @@ def set_argparser(parser):
         parser.add_argument("--latest-limit", dest='latest_limit', type=int,
                              help=_('show N latest packages for a given name.arch'
                                     ' (or latest but N if N is negative)'))
+        parser.add_argument("--disable-modular-filtering", action="store_true",
+                            help=_("list also packages of modules that aren't enabled"))
```

Comment:
..that aren't enabled nor default, or default if another stream is enabled.

Well, this definition is terrible. But maybe we could have "list also packages from non-active module streams" in the help message and explanation of what "active stream" is in the documentation.